discovery_child_development/pipeline/models/taxonomy_classifier/02a_sentence_embeddings.py

```diff
@@ -0,0 +1,66 @@
+"""
+Embed the GPT labelled abstracts using the sentence-transformers library for the simple classifiers.
+--------------
+
+For the existing dataset of GPT labelled docs
+* create embeddings using HuggingFace sentence transformers and the 'all-MiniLM-L6-v2' model
+* save the document IDs and vectors to a parquet file
+
+Usage:
+
+python discovery_child_development/pipeline/models/taxonomy_classifier/02a_sentence_embeddings.py
+
+"""
+from nesta_ds_utils.loading_saving import S3 as nesta_s3
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+from time import time
+
+from discovery_child_development import logging, S3_BUCKET, config, taxonomy_config
+from discovery_child_development.getters import taxonomy_classifier
+
+model = SentenceTransformer(taxonomy_config["sentence_embeddings_model"])
+
+INPUT_DATA_PATH = taxonomy_config["s3_data_path"]
+VECTORS_OUT_PATH = taxonomy_config["s3_vectors_path"]
+VECTORS_FILENAME = taxonomy_config["s3_vectors_name"]
+
+
+def embed_texts(
+    split="train",
+    s3_bucket=S3_BUCKET,
+    vectors_out_path=VECTORS_OUT_PATH,
+    vectors_filename=VECTORS_FILENAME,
+):
+    labelled_text_df = taxonomy_classifier.get_training_data(split=split)
```

Comment:
This was throwing an error, so I changed to
```suggestion
    labelled_text_df, _ = taxonomy_classifier.get_training_data(split=split)
```

Reply:
😶 thank you!